microservices-modules/micronaut/src/main/java/com/baeldung/micronaut/httpfilters/filters/HelloFilter.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.micronaut.httpfilters.filters;
+
+import com.baeldung.micronaut.httpfilters.service.LogService;
+
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.annotation.RequestFilter;
+import io.micronaut.http.annotation.ServerFilter;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+
+@ServerFilter(value="/hello")
```

Comment:
The typical convention is to omit the parameter name when an annotation is given a single value
```suggestion
@ServerFilter("/hello")
```